Add display_override manifest member tabbed value (#34453)
mdn/content: files/en-us/web/manifest/display_override/index.md

    </tr>
  </thead>
  <tbody>
    <tr>
      <td><code>browser</code></td>
      <td>
        The application opens in a conventional browser tab or new window,
        depending on the browser and platform. This is the default.
      </td>
    </tr>
    <tr>
      <td><code>fullscreen</code></td>
      <td>
        All of the available display area is used and no user agent
        {{Glossary("chrome")}} is shown.
      </td>
    </tr>
    <tr>
      <td><code>minimal-ui</code></td>
      <td>

>         The application will look and feel like a standalone application, but
>         will have a minimal set of UI elements for controlling navigation. The
>         elements will vary by browser.

Reviewer: ```suggestion
        The application will look and feel like a standalone application with
        a minimal set of UI elements for controlling navigation. The
        elements will vary by browser.
```

      </td>
    </tr>
    <tr>
      <td><code>tabbed</code></td>
      <td>

>         The application can contain multiple application contexts inside a single OS-level window. Supporting browsers can choose to handle the display of these contexts in whatever way they want, but a common approach is to provide a tab bar to switch between them.

Reviewer: ```suggestion
        The application can contain multiple application contexts inside a single OS-level window. Supporting browsers can choose how to display these contexts, but a common approach is to provide a tab bar to switch between them.
```
